marketing/blog.md

```diff
@@ -2,12 +2,15 @@
 
 ### Blogging Process
 
-When adding content to our blog we follow a similar process as when making changes to our other codebases, with a few minor changes. The process is as follows:
+Content creation and blogging are an effective way to communicate with our community. The process for publishing a blog is as follows:
 
 1. Raise an issue for the content in our private [Github repository](https://github.com/flowforge/content).
 2. If the content is date specific create an 'all day' event on the Google Calendar [FlowForge Publishing Schedule](https://calendar.google.com/calendar/u/0?cid=Y18yMGFjMmM5MmMwYmE0YTYwNDg4NDE1MjBmMGU2YWE0MGFhZGUxNTlkNThjZGY0ZGMwMjA0NTI4ZjFjMTcxZmQ0QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20), include a link to the issue in the event description. If you don't have the permission to create events please ask our Google Workspace admin to give you access.
 3. If you want to work on the content immediately please triage the issue into our [development board](https://github.com/orgs/flowforge/projects/1/views/33).
-4. Create a new branch of the website repository and then follow our standard [development processes](https://flowforge.com/handbook/development/#development-board).
+4. Create the draft blog post in a Google Doc. The document shoud be stored in the [Marketing -> Content](https://drive.google.com/drive/folders/1sdyVfD29dFE74i7zfmL0Cp9tPmPyK0-8?usp=share_link) folder. Everyone at FlowwForge should be given access to comment on the document.  Add a link to the document in the github issue.
+5. Ask people to review the document draft by sharing it directly with them using the Share button in the document.  This will send your reviewers an email notification. 
```

Comment:
```suggestion
5. Ask people to review the document draft by sharing it directly with them using the Share button in the document. This will send your reviewers an email notification. 
```